src/tests/ie_test_utils/common_test_utils/tests/graph_comparator_tests.cpp

```diff
@@ -412,50 +415,74 @@ TEST(GraphComparatorTests, CheckTensorIteratorPositive) {
         function = function_ref->clone();
     }
     comparator.enable(FunctionsComparator::NODES);
+    comparator.enable(FunctionsComparator::SUBGRAPH_DESCRIPTORS);
     auto res = comparator.compare(function, function_ref);
     ASSERT_TRUE(res.valid) << res.message;
 }
 
+namespace {
+std::shared_ptr<ov::Model> make_CheckLoop_model(bool different_body) {
```

Comment:
```suggestion
std::shared_ptr<ov::Model> make_check_loop_model(bool different_body) {
```
Detail to be consistent regarding using `snake_case` or `CamelCase` not mix them.

Reply:
Well .. it looks odd, I admit. I decided to do it this as `CheckLoop` is own name of a test. Let me change it to avoid confusion.